astropy/coordinates/tests/test_intermediate_transformations.py

```diff
@@ -128,6 +129,21 @@ def test_icrs_gcrs_dist_diff(gframe):
                         atol=1e-10*u.pc)
 
 
+def test_non_geocentric_gcrs():
+    """
+    Test transformation to non-geocentric GCRS frames for nearby objects.
+    """
+    t = Time('1980-03-25 00:00')
+    gcrs_frame = GCRS(obstime=t,
+                      obsgeoloc=u.Quantity([-1463969.30185172,
+                                            -5166673.34223433,
+                                            3434985.71204565])*u.m)
+    icrs_coo = ICRS(ra=184.285*u.deg, dec=-1.8802*u.deg, distance=148101810.2*u.km)
+    gcrs_coo = icrs_coo.transform_to(gcrs_frame)
+    
```

Comment:
Remove the spaces from line 143 (empty lines should not have spaces). This is causing a PEP8 failure on Travis.